src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -157,8 +158,24 @@ protected final boolean mineBlock(@NotNull final BlockPos blockToMine, @NotNull
         //calculate fortune enchantment
         final int fortune = ItemStackUtils.getFortuneOf(tool);
 
-        //get all item drops
-        final List<ItemStack> localItems = BlockPosUtil.getBlockDrops(world, blockToMine, fortune);
+        //check if tool has Silk Touch
+        final boolean silkTouch = ItemStackUtils.hasSilkTouch(tool);
+
+        //create list for all item drops to be stored in
+        List<ItemStack> localItems = new ArrayList<ItemStack>();
```

Comment:
I think we can leave this list final and only do addAll in the logic down there.